test/support/minitest_context.rb

```diff
@@ -0,0 +1,18 @@
+### Extracted from: https://github.com/arenaflowers/minitest-context/blob/master/lib/minitest/context.rb
+### Cannot use the gem because the gemspec dependenct requirement is outdated
```

Comment:
What do you mean those development dependencies are outdated?
Feel free to add the gem

Reply:
The note was regarding the minitest-context gemspec being pinned to an outdated minitest version.

I have now removed this minitest context as it causes wierd things to occur. I have duplicated the test files now and basically all tests are passing. There are only 3 tests failing on `test/missing_cached_test.rb` due to the issues stated on https://github.com/svenfuchs/i18n-active_record/issues/106